content/open-tech-will-save-us/otwsu.toml

```
@@ -0,0 +1,270 @@
+[[episodes]]
+code=20
+youtube="d2LhmF5KyCY"
+date="2022-10-26"
+summary="""
```

```suggestion
code = 20
youtube = "d2LhmF5KyCY"
date = "2022-10-26"
summary = """
```

Same for other entries of course.